### PR TITLE
Feature/jpaley ssd digitization

### PR DESCRIPTION
**Pull request by jonpaley**

### CODE REVIEW

**Comment by lackey32** [comment]
I think I've managed to resolve the differences between how you and I were using sensor vs. plane in the channel map and have main fully merged in now. I tested makessdclust_job.fcl, but haven't tested any of the digitization code. That part should be blind to the changes though.

I do still want to take a closer look through some of this before merging with main -  strip and set aren't being set correctly. The strip number in SSDRawDigit doesn't actually correspond to what you think it does.

File: DetGeoMap/service/DetGeoMapService.cc [hunk 1]
```diff
@@ -15,42 +15,42 @@
...
-  namespace cmap
+  namespace dgmap
   {
     
     //------------------------------------------------------------
-    ChannelMapService::ChannelMapService(const fhicl::ParameterSet& pset,
+    DetGeoMapService::DetGeoMapService(const fhicl::ParameterSet& pset,
 					 art::ActivityRegistry & reg)
     {
+      //      art::ServiceHandle<runhist::RunHistoryService> rhs;
```
Comment (lackey32) [inline DetGeoMap/service/DetGeoMapService.cc:25]: remove commented out line

File: DetGeoMap/service/DetGeoMapService.cc [hunk 2]
```diff
@@ -15,42 +15,42 @@
...
+      fDetGeoMap->SetUseGeometry(pset.get<bool>("UseGeometry"));
     }
     
     //----------------------------------------------------------
-    void ChannelMapService::preBeginRun(const art::Run& )
+    void DetGeoMapService::preBeginRun(const art::Run& run)
     {
-      fChannelMap->SetAbortIfFileNotFound(fAbortIfFileNotFound);      
-      art::ServiceHandle<runhist::RunHistoryService> rhs;
+      //      fDetGeoMap->SetAbortIfFileNotFound(fAbortIfFileNotFound);
```
Comment (lackey32) [inline DetGeoMap/service/DetGeoMapService.cc:49]: remove commented out lines here.

File: Digitization/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  LIBRARY_NAME  Digitization
+  LIBRARIES     SimulationBase
+                Simulation
+		G4Base
+		Geometry
+		RawData
+		ChannelMap
+		ChannelMapService_service
+		art_root_io::TFileService_service
+#                ROOT::Hist
```
Comment (lackey32) [inline Digitization/CMakeLists.txt:21]: remove

File: Digitization/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    Simulation
+    Geometry
+    RawData
+    ChannelMap
+    ChannelMapService_service
+    art_root_io::TFileService_service
+    art::Framework_Core
+    art::Persistency_Common
+    messagefacility::MF_MessageLogger
+#    ROOT::Hist
```
Comment (lackey32) [inline Digitization/CMakeLists.txt:38]: remove

File: G4EMPH/g4gen_job.fcl [hunk 5]
```diff
@@ -24,7 +24,7 @@ source:
 {
   module_type: EmptyEvent
   maxEvents: 10000
-  firstRun:  1293
+  firstRun:  2293
```
Comment (lackey32) [inline G4EMPH/g4gen_job.fcl:27]: Did you intend to change this run number? Will it break things for anyone else using this fcl?

File: DetGeoMap/service/DetGeoMapService.cc [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
-/// \brief  Simple service to provide a channel map configured to the right 
+/// \brief  Simple service to provide a detector channel-to-position map 
 /// \author jpaley@fnal.gov
 //////////////////////////////////////////////////////////////////////////
 
 // EMPHATIC includes
-#include "ChannelMap/service/ChannelMapService.h"
-#include "RunHistory/service/RunHistoryService.h"
+#include "DetGeoMap/service/DetGeoMapService.h"
+//#include "RunHistory/service/RunHistoryService.h"
```
Comment (lackey32) [inline DetGeoMap/service/DetGeoMapService.cc:8]: remove this line

**Review by lackey32 — CHANGES_REQUESTED** [review]
Fix in progress for setting chip, set, and strip. Otherwise I just had some inline comments about removing commented out lines that were left over from the previous version of the module.

File: G4EMPH/g4gen_job.fcl (hunk 5 above)
Comment (jonpaley) [reply G4EMPH/g4gen_job.fcl:27]: This shouldn't break anything, but might surprise people who assume they are generating a Phase 1b run.

**Pull request merged**